src/tags/object/Text.js

```diff
@@ -0,0 +1,34 @@
+// stub file to keep docs for Text object tag
+
+/**
+ * The Text tag shows text that can be labeled. Use to display any type of text on the labeling interface.
+ * You can use `<Style>.htx-text{ white-space: pre-wrap; }</Style>` to preserve all spaces in the text, otherwise spaces are trimmed when displayed.
```

Comment:
is this true for Text or just HyperText?

Reply:
This implies that text written will be transformed from:
`text like this` to `textlikethis` 
but instead I think you mean
`text like      this` is transformed to `text like this` 
or?

Reply:
That's classic html space trimming, I thought that trim is always about leading/trailing spaces, so no explanations needed. Also it collapses multiple spaces into one, so maybe we need some other term.
For `HyperText` that's expected behaviour, so this is helpful for `Text` only I think.

Reply:
Ok that makes sense. I think this is fine!